conf.c

```diff
@@ -481,8 +481,14 @@ conf_set_value_in_file(const char *path, const char *key, const char *value,
 		return false;
 	}
 
+#if defined (__MINGW32__) && (__MINGW32_MAJOR_VERSION < 5)
```

Comment:
Perhaps fix (i.e., move `#if ...` to) create_tmp_file instead?

Reply:
I don't want to merge it as is; we should try to keep `#ifdef`s in utility functions only.